client/devdocs/design/search-collection.jsx

```diff
@@ -61,7 +62,10 @@ export default React.createClass( {
 		searchString = example.type.displayName;
 
 		if ( this.props.component ) {
-			return example.type.displayName.toLowerCase() !== this.props.component.replace( /-([a-z])/g, '$1' );
+			const exampleName = invoke( example, 'type.displayName.toLowerCase' );
+			const componentName = invoke( this, 'props.component.replace', /-([a-z])/g, '$1' );
```

Comment:
these lines were preventing navigating to individual component pages in the devdocs. I snuck in a fix here so I could make sure my stuff was working, but maybe I should create a separate PR just for these lines